csrc/sparse/cutlass/sparse_scaled_mm_entry.cu

```diff
@@ -0,0 +1,82 @@
+#include <cudaTypedefs.h>
+
+#include <c10/cuda/CUDAGuard.h>
+#include <torch/all.h>
+
+#if defined ENABLE_SCALED_MM_C3X && ENABLE_SCALED_MM_C3X
+void cutlass_scaled_sparse_mm_sm90(torch::Tensor& c, torch::Tensor const& a,
+                                   torch::Tensor const& e,
+                                   torch::Tensor const& b,
+                                   torch::Tensor const& a_scales,
+                                   torch::Tensor const& b_scales,
+                                   c10::optional<torch::Tensor> const& bias);
+#endif
+
+bool cutlass_scaled_sparse_mm_supports_fp8(int64_t cuda_device_capability) {
+  // CUTLASS FP8 kernels need at least
+  //   CUDA 12.0 on SM90 systems (Hopper)
+  //   CUDA 12.4 on SM89 systems (Lovelace)
+
+#if defined CUDA_VERSION
+  if (cuda_device_capability >= 90) {
+    return CUDA_VERSION >= 12000;
+  } else if (cuda_device_capability >= 89) {
+    return CUDA_VERSION >= 12040;
+  }
+#endif
```

Comment:
We don't support `cutlass_scaled_sparse_mm_supports_fp8` on lovelace systems at all and it requires CUDA 12.2:

https://github.com/NVIDIA/cutlass/blob/e1cd8c7866dd6de02b66a89879795e7d7301aacc/examples/62_hopper_sparse_gemm/62_hopper_sparse_gemm.cu#L553-L559

```suggestion
bool cutlass_scaled_sparse_mm_supports_fp8(int64_t cuda_device_capability) {
  // CUTLASS 2:4 sparse FP8 kernels need at least CUDA 12.2 and SM90 (hopper) 
#if defined CUDA_VERSION
  return cuda_device_capability >= 90 && CUDA_VERSION >= 12020;
#endif
```